webapp/src/Controller/API/JudgehostController.php

```diff
@@ -224,7 +224,6 @@ public function updateJudgeHostAction(
             throw new BadRequestHttpException('Argument \'enabled\' is mandatory');
         }
 
-        /** @var Judgehost $judgehost */
```

Comment:
Why are these removed?

Reply:
Because both phpstorm and phpstan know the type already. We can keep them but then they should change to add `|null` (I.e. they were wrong)

Reply:
Then phpstorm has become smarter, but yeah 👍

Reply:
It has indeed. AI is taking over the world